multiqc/modules/hicexplorer/hicexplorer.py

```diff
@@ -0,0 +1,232 @@
+                        '<br><br><b>Self circle</b><br>'
+                        'Self circles are defined as pairs within 25kb with \'outward\' read orientation'
+                        '<br><br><b>Self ligation</b><br>'
+                        'These are read pairs with a restriction site in between that are within 800 bp.'
+        )
+
+        self.add_section(
+            name='Contact distance',
+            anchor='hicexplorer_contact_distance',
+            plot=self.hicexplorer_contact_distance()
+        )
+
+        self.add_section(
+            name='Read orientation',
+            anchor='hicexplorer_read_orientation',
+            plot=self.hicexplorer_read_orientation()
+        )
+
+    def parse_logs(self, f):
+        """Parse a given HiCExplorer log file from hicBuildMatrix."""
+        data = {}
+        for l in f.splitlines():
+            # catch empty lines
+            if len(l) == 0:
+                continue
+            s = l.split("\t")
+            data_ = []
+            # catch lines with descriptive content: "Of pairs used:"
+            for i in s[1:]:
+                if len(i) == 0:
+                    continue
+                try:
+                    i.replace('(', '')
+                    i.replace(')', '')
+                    i.replace(',', '')
+
+                    data_.append(float(i))
+                except:
+                    data_.append(i)
+            if len(data_) == 0:
+                continue
+            if s[0].startswith('short range'):
+                s[0] = 'short range'
+            elif s[0].startswith('same fragment'):
+                s[0] = 'same fragment'
+            data[s[0]] = tuple(data_)
+        return data
+
+    def hicexplorer_basic_statistics(self):
+        """Create the general statistics for HiCExplorer."""
+        data = {}
+
+        for file in self.mod_data:
+            total_pairs = self.mod_data[file]['Pairs considered'][0]
+            data_ = {
+                'Pairs considered': self.mod_data[file]['Pairs considered'][0],
+
+                'Pairs used': self.mod_data[file]['Pairs used'][0] / total_pairs,
+                'Unmapped': self.mod_data[file]['One mate unmapped'][0] / total_pairs,
+                'Min rest. site distance': self.mod_data[file]['Min rest. site distance'][0],
+                'Max rest. site distance': self.mod_data[file]['Max rest. site distance'][0],
+            }
+            data[self.mod_data[file]['File'][0]] = data_
+        headers = OrderedDict()
+
+        headers['Pairs considered'] = {
+            'title': 'Number of pairs',
+            'modify': lambda x: int(x)
+        }
+
+        headers['Pairs used'] = {
+            'title': '% Used pairs',
+            'max': 100,
+            'min': 0,
+            'modify': lambda x: x * 100
+        }
+        headers['Unmapped'] = {
+            'title': '% Unmapped',
+            'max': 100,
+            'min': 0,
+            'modify': lambda x: x * 100
```

Comment:
I like to give colour schemes that reflect the numbers in tables, where possible. For example, high numbers of `Unmapped`  is bad, so `OrRd` may be good to highlight big numbers with a red colour. See the palette names in the docs here: http://multiqc.info/docs/#step-3-adding-to-the-general-statistics-table

Note that for consistency with other modules, I'd actually prefer to have `% Mapped` instead of `Unmapped` here unless you have a strong preference. Should be easy to switch using the lambda.